akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpHeaderParser.scala

```diff
@@ -438,9 +440,18 @@ private[http] object HttpHeaderParser {
     "Cache-Control: no-cache",
     "Expect: 100-continue")
 
-  def apply(settings: HttpHeaderParser.Settings, log: LoggingAdapter)(onIllegalHeader: ErrorInfo ⇒ Unit = info ⇒ throw IllegalHeaderException(info)) =
+  def apply(settings: HttpHeaderParser.Settings, log: LoggingAdapter)(onIllegalHeader: ErrorInfo ⇒ Unit = defaultIllegalHeaderHandler(settings, log)) =
```

Comment:
What about the usage in `HttpServerBluePrint`? Shouldn't we also use the default one there?